enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/VarLengthExpandQueryPlanAcceptanceTest.scala

```diff
@@ -28,56 +28,101 @@ class VarLengthExpandQueryPlanAcceptanceTest extends ExecutionEngineFunSuite wit
     val query = "PROFILE MATCH (a:From {name:'Keanu Reeves'})-[*..4]->(e:To {name:'Andres'}) RETURN *"
     val result = executeWithCostPlannerAndInterpretedRuntimeOnly(query)
 
-    result should havePlanLike(
-      """
-        |Compiler CYPHER 3.3
-        |
-        |Planner COST
-        |
-        |Runtime SLOTTED
-        |
-        |+-----------------------+----------------+------+---------+-----------------+-------------------+----------------------+------------------+----------------------------------+
-        || Operator              | Estimated Rows | Rows | DB Hits | Page Cache Hits | Page Cache Misses | Page Cache Hit Ratio | Variables        | Other                            |
-        |+-----------------------+----------------+------+---------+-----------------+-------------------+----------------------+------------------+----------------------------------+
-        || +ProduceResults       |              0 |    1 |       0 |               0 |                 0 |               0.0000 | anon[37], a, e   |                                  |
-        || |                     +----------------+------+---------+-----------------+-------------------+----------------------+------------------+----------------------------------+
-        || +Filter               |              0 |    1 |       5 |               0 |                 0 |               0.0000 | anon[37], a, e   | a:From; a.name = {  AUTOSTRING0} |
-        || |                     +----------------+------+---------+-----------------+-------------------+----------------------+------------------+----------------------------------+
-        || +VarLengthExpand(All) |              0 |    4 |       8 |               0 |                 0 |               0.0000 | anon[37], a -- e | (e)<-[:*..4]-(a)                 |
-        || |                     +----------------+------+---------+-----------------+-------------------+----------------------+------------------+----------------------------------+
-        || +Filter               |              0 |    1 |       1 |               0 |                 0 |               0.0000 | e                | e.name = {  AUTOSTRING1}         |
-        || |                     +----------------+------+---------+-----------------+-------------------+----------------------+------------------+----------------------------------+
-        || +NodeByLabelScan      |              1 |    1 |       2 |               0 |                 0 |               0.0000 | e                | :To                              |
-        |+-----------------------+----------------+------+---------+-----------------+-------------------+----------------------+------------------+----------------------------------+
-        |""".stripMargin)
+    result.executionPlanDescription() should useOperatorWithText("VarLengthExpand(All)", "(e)<-[:*..4]-(a)")
+    result.executionPlanDescription() should useOperatorWithText("NodeByLabelScan", ":To")
   }
 
```

Comment:
Since we're porting everything to CypherComparisonSupport, does it make sense to write these new tests using that framework from the beginning?

Reply:
I think we should move the tests over, but at the same time it would be nice to disconnect the two work streams